acra-censor/handlers/handlers_util.go

```diff
@@ -251,6 +210,13 @@ func handleSelectColumnPattern(queryNodes, patternNodes []sqlparser.SQLNode) boo
 			}
 			// check other nodes on equal (except SelectExprs)
 		} else if !reflect.DeepEqual(patternNode, queryNode) {
+			if patternOrderBy, ok := patternNode.(sqlparser.OrderBy); ok {
```

Comment:
can we rewrite this piece into one function? something like

`func ifPatternOrderBy(patternNode, queryNode)`

Reply:
actually we can move first and second `if` checks into `matchOrderBy` function

Reply:
It seems that OrderBy is not a separate pattern - it's part of %%VALUE%%. Yes we can move type assertions to matchOrderBy - but we process types of nodes inside pattern handlers - not in comparers

Reply:
It's up to you then.

I'd prefer hiding if statements somewhere inside function.

Reply:
use switch instead if else and then it should be
```
switch patternNode.(type){
case *sqlparser.SelectExprs:
...
case sqlparser.OrderBy:
...
default: reflect.DeepEqual(...)
 }
```
we don't need to compare via deepequal before type check on OrderBy struct

Reply:
Thanks. Usefully.